Project.toml

```
@@ -15,9 +15,16 @@ GDAL = "add2ef01-049f-52c4-9ee2-e494f65e021a"
 GeoFormatTypes = "68eda718-8dee-11e9-39e7-89f7f65f511f"
 GeoInterface = "cf35fbd7-0cd7-5166-be24-54bfbe79505f"
 GeoInterfaceRecipes = "0329782f-3d07-4b52-b9f6-d3137cf03c7a"
+GeoInterfaceMakie = "0edc0954-3250-4c18-859d-ec71c1660c08"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
+[weakdeps]
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
+
+[extensions]
+ArchGDALGeoInterfaceMakie = "Makie"
```

```suggestion
ArchGDALMakieExt = "Makie"
```

I think this is a more correct name

---

The file will need renaming